src/index.html

```diff
@@ -11,7 +11,177 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+        <legend>
+        Personal information
+        </legend>
+        <div>
+          <label for="Surname">
+            Surname:
+          </label>
+          <input id="Surname" type="text"
+          name="Surname"
+          autocomplete="off"
+          required
+          >
+        </div>
+        <div>
+          <label for="Name">
+            Name:
+          </label>
+          <input type="text" name="Name"
+          id="Name"
+          autocomplete="off"
+          >
```

Comment:
```suggestion
          <input type="text"
            name="name"
            id="Name"
            autocomplete="off"
          >
```
<img width="961" alt="Screenshot 2022-11-11 at 10 34 49" src="https://user-images.githubusercontent.com/104769168/201299622-e17e6e44-bd5c-4011-a31d-e65f1c66f3a4.png">